src/data/data01.json

```diff
@@ -150,6 +150,46 @@
             "originalProblem": { "equation": "", "annotation": "Try your own problem" },
             "currentEditor": {},
             "history": [{}]
-        }
+        },
```

Comment:
I'm not sure if example should be part of a dataset - there should be an example in every dataset.